tasks/extensions.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Ensure PostgreSQL database extensions are present
+  postgresql_ext:
+    name: "{{ item.name }}"
+    ca_cert: "{{ item.ca_cert | default(omit) }}"
+    cascade: "{{ item.cascade | default('no') }}"
+    db: "{{ item.db }}"
+    login_host: "{{ item.login_host | default('localhost') }}"
+    login_password: "{{ item.login_password | default(omit) }}"
+    login_user: "{{ item.login_user | default(postgresql_user) }}"
+    login_unix_socket: "{{ item.login_unix_socket | default(postgresql_unix_socket_directories[0]) }}"
+    port: "{{ item.port | default(omit) }}"
+    schema: "{{ item.schema | default(omit) }}"
+    session_role: "{{ item.session_role | default(omit) }}"
+    ssl_mode: "{{ item.ssl_mode | default('prefer') }}"
+    state: "{{ item.state | default('present') }}"
+    version: "{{ item.version | default(omit) }}"
```

Comment:
Since this is just a simple wrapper, I would suggest not to hard-code parameters.
This achieves same result, but leaves possibility to use some arguments added to postgresql_ext in the future, without role modification.
```
postgresql_ext: "{{ item }}"
```

Reply:
Updated.  I really took this file from the example of the databases and users task, but if `"{{ item }}"` is the best way, that's cool with me.  Thx for the pointer.